test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -717,9 +717,7 @@ public void CreateDescriptors_BuildsDescriptorsWithInheritedProperties()
         {
             // Arrange
             var errorSink = new ErrorSink();
-
-            // Also confirm isStringProperty is calculated correctly.
```

Comment:
side note: My earlier concern was around "losing" explicit checks of all `TagHelperDescriptor` properties against our expectations. That's covered now (slightly less explicitly but that's :ok:) between the test descriptor comparers and the new Razor unit tests. Note as well that without the descriptor comparer fixes, we weren't actually checking `IsStringProperty` at all :frowning: